src/workloads/execution/EventCollection.yml

```diff
@@ -0,0 +1,107 @@
+SchemaVersion: 2018-07-01
+Owner: Storage Execution
+Description: |
+  Insert documents into an event collection.
+
+GlobalDefaults:
+  Collection: &Collection Collection0
+  Database: &db test
+  MaxPhases: &MaxPhases 4
+
+Actors:
+  - Name: CreateEventCollection
+    Type: RunCommand
+    Threads: 1
+    Phases:
+      OnlyActiveInPhases:
+        Active: [0]
+        NopInPhasesUpTo: *MaxPhases
+        PhaseConfig:
+          Repeat: 1
+          Database: *db
+          Threads: 1
+          Operation:
+            OperationName: RunCommand
+            OperationCommand:
+              create: *Collection
+              clusteredIndex:
+                { key: { _id: 1 }, unique: true, autoAssign: "timestamp" }
+  - Name: InsertWith1Thread
+    Type: Loader
+    Threads: 1
+    Phases:
+      OnlyActiveInPhases:
+        Active: [1]
+        NopInPhasesUpTo: *MaxPhases
+        PhaseConfig:
+          Repeat: 1
+          Database: *db
+          Threads: 1
+          CollectionCount: 1
+          DocumentCount: 1000000
+          BatchSize: 1000
+          Document:
```

Comment:
Each document, with field names + values, should be 156 byes (assuming there is no overhead)